app/js/controllers/foyer/individuForm.js

```diff
@@ -48,23 +47,19 @@ angular.module('ddsApp').controller('FoyerIndividuFormCtrl', function($scope, in
         role: individuRole,
         autresRevenusTnsActiviteType: 'bic',
         microEntrepriseActiviteType: 'bic',
-        autoEntrepreneurActiviteType: 'bic'
+        autoEntrepreneurActiviteType: 'bic',
+        situationsPro: []
     };
 
-    if (true === ($scope.captureRelationConjoint = !! options.captureRelationConjoint)) {
-        $scope.individu.statutMarital = 'mariage';
-    }
-
     var isIndividuParent = IndividuService.isRoleParent(individuRole);
+    $scope.individu = isIndividuParent && _.find($scope.situation.individus, { role: individuRole }) || DEFAULT_INDIVIDU;
```

Comment:
The line below means parts of the code could assign to `DEFAULT_INDIVIDU`!

To be safe, `_.cloneDeep(DEFAULT_INDIVIDU)`.

> The cleaner way to do it would be to deep [freeze](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Object/freeze) `Object.freeze(DEFAULT_INDIVIDU)` and, if in ES6, declare it with `const` and use [`Object.assign`](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Object/assign) rather than an `_`-provided equivalent.